components/Modals/Modal.tsx

```diff
@@ -82,3 +92,10 @@ const CloseModalButton = styled(MdClose)`
   padding: 0;
   z-index: 10;
 `
+
+const ModalButton = styled(SmallFormButton)`
+  background-color: #efb7b7;
```

Comment:
You're using this color also with the `Background` component, so it could be something that can be part of the theme